blackbelt/dependencies.py

```diff
@@ -69,11 +69,8 @@ def check(dep, list_path, licenses_path, dev=False, debug=False):
     dep_name, dep_version = dep
     click.echo('Analyzing the package {0}@{1} ...'.format(dep_name, dep_version))
 
-    if dep_name == '.': # check the `pwd` project deps
-        click.echo('The dependency is a local directory, using whatever is currently in node_modules')
-        project_dir = '.'
-        if not os.path.isdir(os.path.join(project_dir, 'node_modules')):
-            raise click.BadParameter('There are no node_modules to analyze in the project directory')
+    if dep_version == '*': # check the `pwd` project deps
```

Comment:
I'm not sure the comment ("check the `pwd` project deps") is still true if we added support for arbitrary directories